clio-server/src/main/scala/org/broadinstitute/clio/server/service/ServerService.scala

```diff
@@ -163,13 +163,14 @@ object ServerService {
   val RecoveryMaxBulkSize: Long = 1000L
 
   def apply(
-    app: ClioApp
+    app: ClioApp,
+    httpServerDAO: HttpServerDAO
   )(implicit executionContext: ExecutionContext, mat: Materializer): ServerService = {
     new ServerService(
       app.serverStatusDAO,
-      app.httpServerDAO,
       app.persistenceDAO,
```

Comment:
Do you think it'd be a ton of work to scrap `ClioApp` entirely, and just pass the `DAO`s around explicitly? I feel like the whole app setup obfuscates more than it helps.

Reply:
probably not too bad? We'll see!